src/Types/VarCharType.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Types;
+
+use Doctrine\DBAL\Driver\Encodings;
+use Doctrine\DBAL\ParameterType;
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+
+use function sprintf;
+
+class VarCharType extends StringType
```

Comment:
Would it make sense to name it as `AsciiStringType`? I believe it can be used with other column types than `VARCHAR` (e.g. `CHAR`).